core/eolearn/tests/test_eoexecutor.py

```diff
@@ -76,19 +94,31 @@ def test_execution_errors(self):
                     if idx != 3:
                         self.assertFalse('error' in stats, 'Workflow {} should be executed without errors'.format(idx))
                     else:
+                        print(stats, idx)
                         self.assertTrue('error' in stats and stats['error'],
                                         'This workflow should be executed with an error')
 
                 self.assertEqual(executor.get_successful_executions(), [0, 1, 2])
                 self.assertEqual(executor.get_failed_executions(), [3])
 
-    def test_report_creation(self):
-        with tempfile.TemporaryDirectory() as tmp_dir_name:
-            executor = EOExecutor(self.workflow, self.execution_args, logs_folder=tmp_dir_name, save_logs=True)
-            executor.run(workers=10)
-            executor.make_report()
+    def test_execution_results(self):
+        for return_results in [True, False]:
+
+            executor = EOExecutor(self.workflow, self.execution_args)
+            results = executor.run(workers=2, multiprocess=True, return_results=return_results)
 
-            self.assertTrue(os.path.exists(executor.get_report_filename()), 'Execution report was not created')
```

Comment:
Fortet it... I see that the test has been moved. (but it does miss a raise exception case)

Reply:
Yes, they were just moved into visualization subpackage. Because of that overall coverage stays the same.

It would be quite complicated to cover the exception case as well - we would have to uninstall a visualization subpackage at some point. Therefore I would leave as it is.